app/jobs/early_years_progress_emails_job.rb

```diff
@@ -0,0 +1,34 @@
+class EarlyYearsProgressEmailsJob < CronJob
+  self.cron_expression = "0 8 * * *" # Every day at 08:00
+
+  def perform
+    periods_after_submission = [2.months, 5.months]
+    today = Date.today
+
+    periods_after_submission.each do |period_after_submission|
+      previous_date = today - period_after_submission
+
+      # handle when current month has more days then previous month
+      if today.last_day_of_the_month? && (today.day > previous_date.end_of_month.day)
+        next
+      end
+
+      window_open = previous_date.beginning_of_day
+
+      # handle when previous month has more days that current month
+      window_close = if today.last_day_of_the_month? && (previous_date.end_of_month.day > today.day)
+        previous_date.end_of_month.end_of_day
+      else # normal 1:1
+        previous_date.end_of_day
+      end
+
+      Claim
+        .by_policy(Policies::EarlyYearsPayments)
+        .awaiting_decision
+        .where(submitted_at: window_open..window_close)
+        .find_each do |claim|
+          EarlyYearsPaymentsMailer.with(claim:).progress_update.deliver_later
+        end
+    end
+  end
```

Comment:
I'm probably missing something!

Reply:
- someone submits a claim on the `July 31st`
- 2 months later is `September 31st` except that September only has 30 days
- when/how does this person receive their 2 month progress update?

Reply:
OK makes sense!

Reply:
I guess date addition rather than date subtraction would work.

```
scope.where("submitted_at::date + interval '2 months' = ?", Date.today).or(
  scope.where("submitted_at::date + interval '5 months' = ?", Date.today)
)
```
Have approved the PR though 👍